src/main/webapp/app/exercises/shared/exercise/exercise.service.ts

```diff
@@ -55,12 +55,13 @@ export class ExerciseService {
      */
     validateDate(exercise: Exercise) {
         exercise.dueDateError = exercise.releaseDate && exercise.dueDate ? !exercise.dueDate.isAfter(exercise.releaseDate) : false;
-
         exercise.assessmentDueDateError =
-            exercise.assessmentDueDate && exercise.releaseDate
-                ? !exercise.assessmentDueDate.isAfter(exercise.releaseDate)
-                : exercise.assessmentDueDate && exercise.dueDate
-                ? !exercise.assessmentDueDate.isAfter(exercise.dueDate)
+            exercise.releaseDate && exercise.assessmentDueDate
```

Comment:
Maybe it is just me, but I think this could be refactored to an `if-else` chain, this looks more complex than it should be?